src/core/Akka/Actor/ActorPath.cs

```diff
@@ -445,7 +445,7 @@ public string ToStringWithoutAddress()
         /// <inheritdoc/>
         public override string ToString()
         {
-            return ToStringWithAddress();
+            return $"{Address}{Join()}";
```

Comment:
Might need to check performance impact here - making a note

Reply:
should not, the previous `ToStringWithAddress` is doing the same inside